StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -561,7 +561,7 @@ void StPicoDstMaker::fillTracks()
   {
     StMuTrack* pTrk = (StMuTrack*)mMuDst->primaryTracks(i);
     if (!pTrk) continue;
-    if (pTrk->id() < 0 || pTrk->id() >= 50000)
+    if (pTrk->id() < 0)
```

Comment:
In that case, the value of nTrk can be lowered and check could be brought back to show the intention.